src/Javis.jl

```diff
@@ -140,13 +142,47 @@ A SubAction should not be created by hand but instead by using one of the constr
 """
 mutable struct SubAction <: AbstractAction
     frames::Frames
+    anim::Animation
     func::Function
     transitions::Vector{Transition}
     internal_transitions::Vector{InternalTransition}
 end
 
 """
-    SubAction(frames::UnitRange, func::Function)
+    SubAction(frames, easing::Easing, args...)
+
+A `SubAction` can be defined with frames an easing function and either a function or
+transformations.
+In the following example a filled circle with radius 50 appears in the first 20 frames,
+which means the opacity is increased from 0 to 1.0. The interpolation function used here is
+sineio from [`Animations.jl`](https://github.com/jkrumbiegel/Animations.jl).
+Then it stays at full opacity and disappears the same way in the last 20 frames using a
+linear decay.
+
+# Example
+javis(demo, [
+    BackgroundAction(1:100, ground),
+    Action((args...)->circle(O, 50, :fill); subactions = [
+        SubAction(1:20, sineio(), appear(:fade)),
+        SubAction(81:100, disappear(:fade))
+    ])
+])
+
+# Arguments
+- `frames`: A list of frames for which the function should be called.
+    - The frame numbers are relative to the parent [`Action`](@ref).
+- `easing::Easing`: The easing function for `args...`
+- `args...`: Either a function like [`appear`](@ref) or a Transformation
+    like [`Translation`](@ref)
+"""
```

Comment:
Good job on the docstring! I would suggest changing the format and presentation like this:

```mkd
A `SubAction` can be defined with frames, an easing function, and either a function or
transformation(s).

# Arguments
- `frames`: A list of frames for which the function should be called.
    - The frame numbers are relative to the parent [`Action`](@ref).
- `easing::Easing`: The easing function for `args...`
- `args...`: Either a function like [`appear`](@ref) or a Transformation
    like [`Translation`](@ref)

# Example
In the following example a filled circle with radius 50 appears in the first 20 frames,
which means the opacity is increased from 0 to 1.0.
The interpolation function used here is
sineio from [`Animations.jl`](https://github.com/jkrumbiegel/Animations.jl).
Then it stays at full opacity and disappears in the last 20 frames using a linear decay.

```julia
javis(demo, [
    BackgroundAction(1:100, ground),
    Action((args...)->circle(O, 50, :fill); subactions = [
        SubAction(1:20, sineio(), appear(:fade)),
        SubAction(81:100, disappear(:fade))
    ])
])
```


Also, it might be good to also explain what an easing function is and link to the easing functions available at `Animations.jl`.

Reply:
Normally we have Arguments at the end, right? Do you think this order is better or shall I just update the explanation?

Reply:
Hmmm... I missed that we switched around arguments and examples. Now that I think about it, your organization scheme I think is better with Examples first and the Arguments later. Just update the explanation and otherwise, this is fine.

Reply:
Hope the docstring is better now.